metadata/service/client/single.yml

```diff
@@ -4,3 +4,65 @@ parameters:
   grafana:
     client:
       enabled: true
+      base_templates:
+        main_dashboard:
```

Comment:
It is in grafana/meta/grafana.yml. It provides a main dashboard with definitions for base_(dashboards,row,panels). See https://github.com/salt-formulas/salt-formula-grafana/pull/25/files#diff-8e58111375f36749d3dea5fca3957dd3R7